mfr/extensions/pdf/settings.py

```diff
@@ -0,0 +1,9 @@
+from mfr import settings
+
+
+config = settings.child('PDF_EXTENSION_CONFIG')
+
+EXPORT_TYPE = config.get('EXPORT_TYPE', 'pdf')
+EXPORT_MAXIMUM_SIZE = config.get('EXPORT_MAXIMUM_SIZE', '1200x1200')
+EXPORT_EXCLUSIONS = config.get('EXPORT_EXCLUSIONS', ['.pdf', ])
```

Comment:
the image settings is done the same way (its mostly c/p from the image settings). I don't see a method in mfr.settings to handle lists. 
Should I be feeding this a string and making the list myself in the renderer or something like that?

also, should the image extension be changed as well?

Reply:
Other small note: even though the list is turned into a string, it will still find '.pdf' in that string and exclude it. Not sure if its intended to work that way, but that must be why its working for images settings.  (this is why it works even though it seems like it shouldn't)

As mentioned above im open to any fix for this

Reply:
Blarf, sorry I was misremembering. The similar example I was thinking of was [here](https://github.com/CenterForOpenScience/modular-file-renderer/blob/develop/mfr/server/settings.py#L35).  We support passing a list via the env by requiring it to be a space-separated string that gets split upon construction.  And yes, the image renderer settings should be doing it this way, but we never bothered to audit the settings when we set it up.  There are probably lots of settings that need this, so we'll create a separate janitorial ticket to audit and clean these up.

Reply:
Done, and ticket made.